docs/requirements.txt

```diff
@@ -1,4 +1,6 @@
 sphinx
 sphinxcontrib-napoleon
+sphinx_rtd_theme
 recommonmark 
 sphinx-markdown-tables
+cython
```

Comment:
Here are the `docs` python requirements, the cython is required in compiling cvm package.
So move the cython keyword to file `install/requirements.txt`.

Another considerable thing:  it seems to be better to remove the complicated packages in the 
`install/requirements.txt` like mxnet and gluoncv since those packages depend on the different environments
in various machines. You can introduce how to install mxnet in ubuntu and macOS in an extra markdown file.

Reply:
Already updated. These two packages have official installation instructions given in the official sites.
Another doc would be redundant. So I include the sites in `MRT Installation` -> `1. Install Python`